conda/faiss/meta.yaml

```diff
@@ -65,7 +65,7 @@ outputs:
         - make  # [not win]
       host:
         - python {{ python }}
-        - numpy 1.11.*
+        - numpy >=1.11.*
```

Comment:
We need to build against numpy `1.11`, as this is the earliest version of numpy we support, and we use the C API. If we relax the constraint here, packages generated will only be compatible with numpy versions newer than whichever conda-build ends up selecting at build time.

Reply:
I see. I confirmed that you fixed this part on #1894, so I've reverted this change.